app/controllers/admin/missions_controller.rb

```diff
@@ -8,7 +8,9 @@ def new
     @mission = @session.missions.new
   end
 
-  def edit; end
+  def edit
+
+ end
```

Comment:
**[checkstyle]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Lint/DefEndAlignment: `end` at 13, 1 is not aligned with `def` at 11, 2.